libsrc/target/aquarius/interrupt/im1_init.asm

```diff
@@ -0,0 +1,64 @@
+        SECTION code_clib
+        PUBLIC  _im1_init
+        PUBLIC  im1_init
+
+        EXTERN  asm_interrupt_handler
+        EXTERN  im1_vectors
+        EXTERN  l_pop_ei
+        EXTERN  l_push_di
+        extern  tick_count_isr
+
+        defc    Z80_OPCODE_JP=$c3
+        defc    EXISTING_INT_SIZE=3
+        defc    IO_IRQMASK=$ee
```

Comment:
The defc from here could go into aqplus.def

Reply:
I've been talking to the AQ+ guys. There are two inc files I think we should pull in from their source base which will handle all of the Aq+ specific devices. sbasic.inc and plus.inc, where is the best place to put these so they can be used by z88dk and developer code?

Reply:
Plonk them in lib/target/aquarius/def Then the `OPTIONS` line in aquarius.cfg can have `-Ca-IDESTDIR/lib/target/aquarius/def` added to it - that way it's just `INCLUDE "plus.inc"` in files.